### PR TITLE
Add readme file

### LINKED ISSUE #12: Add README for the community repository
As an Eiffel Community Member, I would like a README file created for the community repository so that I know what Eiffel Community is about and can find the links to guidelines, charted, and process documents.

### PR DESCRIPTION
**Pull request by henning-roos**
### Applicable Issues
#12 

### Description of the Change
 * Add README.md
 * Add logo

### Alternate Designs
-

### Benefits
Every repository in eiffel community requires a README file. Community is no exception.

### Possible Drawbacks
-

### Sign-off
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Henning Roos henning.roos@ericsson.com

### CODE REVIEW

**Comment by henning-roos** [comment]
If you have trouble visualizing the changes you can go here: https://github.com/henning-roos/community/tree/add_readme

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-TBD
+<img src="./images/community-logo.png" alt="Eiffel Community Repository Template" width="350"/>
+
+# Eiffel Community
+This repository contains common Eiffel resources and information that and not does belong to a specific Eiffel product. General issues that concerns all repositories can be posted and discussed in this repository as well.
```
Comment (e-pettersson-ericsson) [inline README.md]: The first sentence reads a bit funny, maybe it should be "... that does not ..."

**Comment by henning-roos** [comment]
@eiffel-community/eiffel-community-maintainers any more comments before I merge?

**Comment by sodero** [comment]
Looks good to me.

**Comment by henning-roos** [comment]
> please avoid using words like "product".

Thanks for the comment!

I can change "product" to "repository" or is there a better word? "Tool"?

Was there anything else you refer to when you write: 'words like "product"'? Or was it just "product" itself?

**Comment by fdegir** [comment]
I think repository is a better word.

It was about just "product" itself. If I see some other words similar to product, I'll create an issue or better a PR.

**Pull request merged**